FRW/Modules/Browser/View/BrowserAuthnViewModel.swift

```diff
@@ -8,44 +8,60 @@
 import Flow
 import SwiftUI
 
+// MARK: - BrowserAuthnViewModel.Callback
+
 extension BrowserAuthnViewModel {
     typealias Callback = (Bool) -> Void
 }
 
+// MARK: - BrowserAuthnViewModel
+
 class BrowserAuthnViewModel: ObservableObject {
-    @Published var title: String
-    @Published var urlString: String
-    @Published var walletAddress: String?
-    @Published var logo: String?
-    @Published var network: Flow.ChainID?
-    private var callback: BrowserAuthnViewModel.Callback?
+    // MARK: Lifecycle
 
-    init(title: String,
-         url: String,
-         logo: String?,
-         walletAddress: String?,
-         network: Flow.ChainID? = nil,
-         callback: @escaping BrowserAuthnViewModel.Callback)
-    {
+    init(
+        title: String,
+        url: String,
+        logo: String?,
+        walletAddress: String?,
+        network: Flow.ChainID? = nil,
+        callback: @escaping BrowserAuthnViewModel.Callback
+    ) {
         self.title = title
-        urlString = url
+        self.urlString = url
         self.logo = logo
         self.network = network
         self.walletAddress = walletAddress
         self.callback = callback
     }
 
+    deinit {
+        callback?(false)
+        WalletConnectManager.shared.reloadPendingRequests()
+    }
+
+    // MARK: Internal
+
+    @Published
+    var title: String
+    @Published
+    var urlString: String
+    @Published
+    var walletAddress: String?
+    @Published
+    var logo: String?
+    @Published
+    var network: Flow.ChainID?
```

Comment:
Is it the linter suggesting to split this way, or it's a style that we're committing to use? 
Asking because I think in many cases having everything in one line makes the code more compact and easier to read

Reply:
This is the result of swiftformat formatting, not manual
@jeden